From 687d9fb9e3de832379680e9d5268331011c92afa Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 26 Jan 2023 08:53:26 -0800
Subject: [PATCH] drd/tests: Include missing <cstdint>

gcc 13 moved some includes around and as a result <cstdint> is no longer
transitively included [1]. Explicitly include it for uint{32,64}_t.

Fixes
tsan_thread_wrappers_pthread.h:91:9: error: 'int64_t' does not name a type

[1] https://gcc.gnu.org/gcc-13/porting_to.html#header-dep-changes

Upstream-Status: Submitted [https://bugs.kde.org/show_bug.cgi?id=464859]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 drd/tests/tsan_thread_wrappers_pthread.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drd/tests/tsan_thread_wrappers_pthread.h b/drd/tests/tsan_thread_wrappers_pthread.h
index f15e6ad..4cc8062 100644
--- a/drd/tests/tsan_thread_wrappers_pthread.h
+++ b/drd/tests/tsan_thread_wrappers_pthread.h
@@ -55,6 +55,7 @@
 #define NO_TLS
 #endif
 
+#include <cstdint>
 #include <string>
 using namespace std;
 
-- 
2.39.1

